functions-methods.js

```diff
@@ -20,6 +52,42 @@
 // typeOfEmail("novi.nlaapjesk@outlook.com") geeft geeft "Extern" <-- deze moet het ook doen!
 // typeOfEmail("a.wiersma@outlook.com") geeft "Extern"
 
+/* Stap 1 : Array opmaken emails
+   Stap 2 Functie declareren
+   stap 3: een variabelen die de uitkomst weergeeft.
+   Stap 4 : statements opschrijven die checkt of het een medewerker is. met de stringmethode
+       if : -Loop door de array met emailadressen en check of novi-education erin staat om de boolean Student op true te laten komen
+            -Als boolean true is, geef bericht Student.
+       if: - emailadressen check op @novi.nl erin staat
+            - Als boolean true is: output Medewerker
+       if: - emailadressen check op @novi.nl erin staat
+            - Als boolean true is: output : extern
+       else: extern output
+ */
+
+
+let emailAdress = ["n.eeken@novi-education.nl", "t.mellink@novi.nl", "novi.nlaapjesk@outlook.com", "a.wiersma@outlook.com"];
+
+
+function typeOfEmail (email) {
+    for (let i = 0; i < emailAdress.length; i++) {
+        if (emailAdress[i].includes("@novi-education")) {
+            console.log("Dit is een student emailadres");
+        }
+
+        if (emailAdress[i].includes("@novi.nl")) {
+            console.log("Dit is een medewerker emailadres");
+        }
+
+        if (emailAdress[i].includes("@outlook.com")) {
+            console.log("Dit is een extern emailadres");
+        }
```

Comment:
Ik zou er wel `else-if`'s van maken:
```suggestion
        if (emailAdress[i].includes("@novi-education")) {
            console.log("Dit is een student emailadres");
        } else if (emailAdress[i].includes("@novi.nl")) {
            console.log("Dit is een medewerker emailadres");
        } else if (emailAdress[i].includes("@outlook.com")) {
            console.log("Dit is een extern emailadres");
        }
```